### ISSUE TITLE
Supply a better message if the course doesn't exist

### ISSUE THREAD

**Issue by jonespm** [tl-its-umich-edu/my-learning-analytics#670]
Thank you for contributing to this project!

- Make sure to search the [issues](https://github.com/tl-its-umich-edu/my-learning-analytics/issues) for duplicates first!

## Expected behavior (A clear and concise description of what you expected to happen) : 
If you go to a course that doesn't exist for instance
/courses/99999999 it should display a message that the course doesn't exist.
## Describe the bug (Tell us what happens instead of the expected behavior) :
Currently the UI just says "Something went wrong" which is sometimes hard to find what the error is
![image](https://user-images.githubusercontent.com/27447/63454759-6937c580-c419-11e9-9514-3e0b78f6674b.png)

## Steps to Reproduce :
1. Just go to a URL that doesn't exist in the system
2. See the error in the UI

**Comment by jonespm** [comment]
There's also an issue if the user doesn't have permission to the course. The api just throws a 403 but the frontend displays the same message. Need to figure out a good way of dealing with these.

**Comment by luctiger98** [comment]
Hello, myself and a partner are currently students enrolled in EECS 481. We recently saw a presentation regarding MyLA in class and wanted to contribute to fixing an open source issue. We just wanted to let you know that we will be attempting to tackle this bug. We will return with further developments and questions if necessary.

**Comment by justin0022** [comment]
Thanks @luctiger98, we appreciate your help! Don't hesitate to ask us if you have any questions.

**Comment by jacobmal** [comment]
Hello, me and @luctiger98 have been working on this bug and we think we've solved it. Now, if a user accesses a course that is not found (404) they will see:
![404-Not_Found](https://user-images.githubusercontent.com/46611530/69679999-a1cde780-1077-11ea-8b55-cd415adf8a76.png)


And if they access a course that they do not have permission to view (403) they will see:
![403-Forbidden](https://user-images.githubusercontent.com/46611530/69680881-4ea96400-107a-11ea-9641-92bf553eaaf4.png)


Before we submitted a pull request with the changes we had a couple of questions. First, are these descriptions given for the error messages descriptive enough?

Also, while searching for a solution for this bug, I found that there is no 404 handler for links that do not exist. This is what shows up when I try to open a non existent link with Django Debug options off: 
![Not-Found-Link](https://user-images.githubusercontent.com/46611530/69686086-93d69180-108c-11ea-8026-15037478d522.png)

Would creating a Django 404 handler with similar styling as the previous errors be a helpful feature to add?
